account_credit_control/README.rst

```diff
@@ -18,10 +23,14 @@ applied on all partners in each company, under the Accounting tab.
 
 You are able to specify a particular policy for one partner or one invoice.
 
+.. figure:: path/to/local/image.png
+   :alt: alternative description
+   :width: 600 px
```

Comment:
Does it comes from the template ? You probably don't want that here. As we don't have an image.